docs/provider.md

```diff
@@ -34,6 +34,55 @@ We have tested using the available policy `AmazonRoute53FullAccess` however it s
 
 [https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/access-control-managing-permissions.html](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/access-control-managing-permissions.html)
 
+By default, Kuadrant will list the available zones and find the matching zone based on the listener host in the gateway listener. If it finds more than one matching zone for a given listener host, it will not update any of those zones. 
+When providing a credential you should limit that credential down to just have write access to the zones you want Kuadrant to manage. Below is an example of a an AWS policy for doing this type of thing:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "route53:ListTagsForResources",
+                "route53:GetHealthCheckLastFailureReason",
+                "route53:GetHealthCheckStatus",
+                "route53:GetChange",
+                "route53:GetHostedZone",
+                "route53:ChangeResourceRecordSets",
+                "route53:ListResourceRecordSets",
+                "route53:GetHealthCheck",
+                "route53:UpdateHostedZoneComment",
+                "route53:UpdateHealthCheck",
+                "route53:CreateHealthCheck",
+                "route53:DeleteHealthCheck",
+                "route53:ListTagsForResource",
+                "route53:ListHealthChecks",
+                "route53:GetGeoLocation",
+                "route53:ListGeoLocations",
+                "route53:ListHostedZonesByName",
+                "route53:GetHealthCheckCount"
+            ],
+            "Resource": [
+                "arn:aws:route53:::hostedzone/Z08187901Y93585DDGM6K",
+                "arn:aws:route53:::healthcheck/*",
+                "arn:aws:route53:::change/*"
+            ]
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "route53:ListHostedZones"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
```

Comment:
Presumably after we drop support for provider health checks we won't really need those HealthCheck permissions? Should we keep them?

Reply:
I will add a note to the health check epic to update the needed perms